packages/react/react/src/hooks/useDidUpdate.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import { useState } from 'react';
+import { useDidUpdate } from './useDidUpdate';
+
+const TOSS_TEXT = 'TOSS';
+
+interface ITestComponentProps {
```

Comment:
We usually do not use the `I` prefix for interfaces in our repository. Why not we go on with `TestComponentProps`?